packages/front-end/hooks/useMembers.ts

```diff
@@ -0,0 +1,19 @@
+import { useMemo } from "react";
+import useUser from "./useUser";
+
+export default function useMembers() {
+  const { users } = useUser();
+
+  const memberUsernameOptions = useMemo(() => {
+    const memberUsernameOptions = [];
+    for (const user of users.values()) {
+      memberUsernameOptions.push({
+        display: user.name ? user.name : user.email,
+        value: user.name ? user.name : user.email,
+      });
+    }
+    return memberUsernameOptions;
+  }, [users]);
+
+  return { memberUsernameOptions };
```

Comment:
Should I move this functionality to the `useUser` hook?

Reply:
I think this is fine for now.